app/templates/gentelella/guest/event/schedule.html

```diff
@@ -151,15 +151,16 @@ <h1 style="font-weight: 300; font-size: 24px">
                     $('.overlay').addClass('visible');
                     $('.btn-pop-out').removeClass('fa-expand').addClass('fa-compress');
                     var width = $("#timeline").width() - $('.timeunits.x1').width();
-                    $('.microlocation-container').css("width", width + "px");
+                    $('.microlocation-container').css("max-width", width + "px");
                     $('.session.scheduled').popover('hide');
                 }
                 function close(){
                     $scheduler.removeClass('scheduler-pop');
                     $body.removeClass('noscroll');
                     $('.overlay').removeClass('visible');
                     $('.btn-pop-out').addClass('fa-expand').removeClass('fa-compress');
-                    $('.microlocation-container').css("width", "fit-content");
```

Comment:
As far as I remember, the fit-content style was adding to avoid the extra space on the right side in case the number of microlocations were less. Can you test that it still works fine on making your change?